seed/templates/seed/_sidebar.html

```diff
@@ -29,6 +29,10 @@
             <div class="item_name" translate>Organizations</div>
             <div ng-show="logged_in" class="badge badge_menu">{$ organizations_count | number:0 $}</div>
         </a>
+        <a id="sidebar-accounts" ng-class="{item: true, active: is_active('/analyses/' + menu.user.organization.id), 'disabled-item': !logged_in}" title="{$:: 'Analyses' | translate $}" href="{% url "seed:home" %}#/analyses/{$:: menu.user.organization.id $}">
```

Comment:
I'm not sure how to fix this, but it looks like the org id isn't being updated when I change update it so the analyses link always goes to the analyses for the same org until I refresh the page.

![Screen Shot 2021-09-15 at 12 14 31 PM](https://user-images.githubusercontent.com/18518728/133487295-1107d10a-b2dc-4a8e-96e2-1656ea5e35d2.png)


I think there's some kind of refresh event happening when the user updates the selected org, not sure how to integrate it with the sidebar though.

I think we also need to handle the case where the user is on the analyses page and they change the active org (it should refresh the analyses page for the new org). Again this is handled by some kind of refresh on other pages (e.g. try going to a property detail page then changing the org) but I don't know much else about it.

Reply:
Good catch.  I'll take a look!

Reply:
I've refactored the analyses pages to update when the org changes.  The org ID is no longer a URL param and the link behaves like the other nav links, getting org id from the user_service instead.